### PR TITLE
loadbalancer additional_vips #2699

### LINKED ISSUE #2699: Missing additional vips in octavia loadbalancer sdk
Hello, new feature in octavia loadbalancer (additional_vips) are not implemented in gophercloud.
Taking the point.

https://github.com/gophercloud/gophercloud/blob/master/openstack/loadbalancer/v2/loadbalancers/results.go#L16
https://github.com/gophercloud/gophercloud/blob/master/openstack/loadbalancer/v2/loadbalancers/requests.go#L80

https://docs.openstack.org/api-ref/load-balancer/v2/?expanded=show-load-balancer-details-detail#show-load-balancer-details

New in version 2.26

### PR DESCRIPTION
**Pull request by snigle**
Fixes #2699

https://docs.openstack.org/api-ref/load-balancer/v2/?expanded=show-load-balancer-details-detail#show-load-balancer-details

### CODE REVIEW

File: openstack/loadbalancer/v2/loadbalancers/requests.go [hunk 1]
```diff
@@ -140,6 +140,10 @@ type CreateOpts struct {
 
 	// Tags is a set of resource tags.
 	Tags []string `json:"tags,omitempty"`
+
+	// The additional ips of the loadbalancer. Subnets must all belong to the same network as the primary VIP.
+	// New in version 2.26
+	AdditionalVIps []AdditionalVip `json:"additional_vips,omitempty"`
```
Comment (EmilienM) [inline openstack/loadbalancer/v2/loadbalancers/requests.go:146]:
```suggestion
	AdditionalVips []AdditionalVip `json:"additional_vips,omitempty"`
```

File: openstack/loadbalancer/v2/loadbalancers/results.go [hunk 2]
```diff
@@ -77,6 +77,16 @@ type LoadBalancer struct {
 	// Tags is a list of resource tags. Tags are arbitrarily defined strings
 	// attached to the resource.
 	Tags []string `json:"tags"`
+
+	// The additional ips of the loadbalancer. Subnets must all belong to the same network as the primary VIP.
+	// New in version 2.26
+	AdditionalVIps []AdditionalVip `json:"additional_vips"`
```
Comment (EmilienM) [inline openstack/loadbalancer/v2/loadbalancers/results.go:83]:
```suggestion
	AdditionalVips []AdditionalVip `json:"additional_vips"`
```

File: openstack/loadbalancer/v2/loadbalancers/testing/fixtures.go [hunk 3]
```diff
@@ -287,7 +289,12 @@ var (
 		ProvisioningStatus: "PENDING_CREATE",
 		OperatingStatus:    "OFFLINE",
 		Tags:               []string{"test", "stage"},
-	}
+		AdditionalVIps: []loadbalancers.AdditionalVip{
```
Comment (EmilienM) [inline openstack/loadbalancer/v2/loadbalancers/testing/fixtures.go:292]:
```suggestion
		AdditionalVips: []loadbalancers.AdditionalVip{
```

**Comment by snigle** [comment]
> Please also add testing coverage in acceptance. Thanks!

tried to write them but I don't have environment to execute it

**Comment by EmilienM** [comment]
CI seems unhappy now with

```
ERROR octavia.network.drivers.neutron.allowed_address_pairs [None req-e3045032-a87e-405a-9ebd-22444b123442 admin admin] Error creating neutron port on network fe1b4794-ce05-4b25-9fa9-7aa5f3ca23b7 due to ConflictException('ConflictException: 409').: openstack.exceptions.ConflictException: ConflictException: 409: Client Error for url: http://10.1.0.39:9696/networking/v2.0/ports, IP address 192.168.2.127 already allocated in subnet 80595bf8-fe08-485f-a1b3-1f03ea35e965
```

**Comment by EmilienM** [comment]
Could you please rebase, so it'll trigger CI again and if green I'll merge it.

**Comment by mdbooth** [comment]
@dulek Any thoughts on this?

**Comment by dulek** [comment]
It looks good to me from the functional standpoint.

**Comment by EmilienM** [comment]
please rebase, it'll trigger proper CI jobs. Thanks !

**Pull request closed without merging**